### DISCUSSION TITLE
GitHub copilot for ServiceNow

### DISCUSSION THREAD

**Discussion by sharat200** [community/community#110735] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, anyone in the community have tried using GitHub copilot for ServiceNow platform. If yes, can you please share how to do it and your experience/feedback.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sharat200 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by sharat200** [reply]
Thank you, I appreciate the help.
